app/forms/concerns/wizard.rb

```diff
@@ -48,7 +48,7 @@ def state
   end
 
   def state_excluded_attributes
-    %w[state_store errors validation_context]
+    %w[state_store errors validation_context context_for_validation]
```

Comment:
Out of curiosity, what was the issue here? 🤔

Reply:
Rails 8 added a new attribute that our wizard forms need to ignore. Otherwise the wizard will complain that it doesn't know what to do with `context_for_validation`.

The wizard doesn't need this attribute to work. It will try to save it to redis and then to Active record which we don't want https://github.com/DFE-Digital/apply-for-teacher-training/blob/c74610680b464531cb4360ba34bfce16eef3d12d/app/forms/concerns/wizard.rb#L32